docs/source/deep_dives/comet_logging.rst

```diff
@@ -0,0 +1,49 @@
+.. _comet_logging:
+
+================
+Logging to Comet
+================
+
+This deep-dive will guide you through how to set up logging to Comet in torchtune.
+
+.. grid:: 1
+
+    .. grid-item-card:: :octicon:`mortar-board;1em;` What this deep-dive will cover
+
+      * How to get started with Comet
+      * How to use the :class:`~torchtune.utils.metric_logging.CometLogger`
+      * How to log configs, metrics, and model checkpoints to Comet
+
+Torchtune supports logging your training runs to `Comet <https://www.comet.com/site/?utm_source=torchtune&utm_medium=docs&utm_content=docs>`_.
+An example Comet workspace from a torchtune fine-tuning run can be seen in the screenshot below.
+
+.. image:: ../_static/img/comet_torchtune_project.png
```

Comment:
I think you need to grab this png from the original PR, unless it needs to be updated @Lothiraldan?

Reply:
I will regenerate a new screenshot after re-running the example with the latest version of this PR, that's on me